mail/mailutils/Portfile

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                mailutils
+version             3.6
+revision            0
+
+categories          mail
+platforms           darwin
+license             GPL-3
+
+maintainers         nomaintainer
+
+description         General-Purpose Mail Packages
+long_description    Mailutils is a swiss army knife of electronic mail handling. \
+                    It offers a rich set of utilities and daemons for processing e-mail.
+
+homepage            https://mailutils.org/
+master_sites        gnu:mailutils
+use_xz              yes
+
+checksums           rmd160          6f222222b6337afe6014d6af16d6e4bcf6884009 \
+                    sha256          46e23b53caba26b55c1dfd4e012047742bf6d39434c72d4da7ac4d9bf9fdf01e \
+                    size            2813412
+
+configure.args      --without-readline \
+                    --with-ldap=no \
+                    --enable-build-servers=no \
+                    --with-guile=no \
+                    --without-gnutls \
+                    --without-fribidi \
+                    --with-tcp-wrappers=no \
+                    --disable-cxx \
+                    --disable-python
+
+depends_build       port:coreutils \
+                    port:gawk \
+                    port:grep \
+                    port:gsed \
+                    port:m4
+
+variant readline description {Enable readline} {
+    configure.args-delete   --without-readline
```

Comment:
Please try to consistently use something like
```
configure.args-replace --without-readline --with-readline
```
for all variants. Default values may change in the future.

Reply:
i am going to apply this change right away. i wasn't aware of the -replace: there are sooo many things i still have to learn :) 

i am also going to try to squash / rebase against master using the "trick" you suggested.

btw, thank you very much for the amazing euphemism about my being not confortable with merge/rebase :)

i am currently going throughtout lot of macport's and (especially) git's documentation to avoid pestering all of you with silly mistakes, but the real truth is that i am a clumsy old newbie (yet having a lot of fun myself) :-)

cheers,
\\--
ferdy